controllers/spec/enforcer/resources_count_spec/statefulset/ReplicaDbCountSpecEnforcer.go

```diff
@@ -149,10 +154,14 @@ func (r *ReplicaDbCountSpecEnforcer) getNbreDeployedReplicas() int32 {
 func (r *ReplicaDbCountSpecEnforcer) getExpectedNbreReplicasToDeploy() int32 {
 	expectedNbreToDeploy := r.resourcesStates.StatefulSets.SpecExpectedNbreToDeploy
 
+	if r.isStandbyEnabled() {
+		return expectedNbreToDeploy
```

Comment:
>This is the expected number of replicas regardless if it is standby or not. Isn't it?

It isn't. It is the number of total instances running, primary + replicas.

This is the Replica enforcer, if you check below, in case of non-standBy configs, it subtracts the primary to the configured number of instances.

>If so does it make sense to have more than one replica in the standBy cluster?

I think so, the same sense it makes to have more than one replica in a non-standBy.